### PR TITLE
Adding some info to the users table

### PR DESCRIPTION
**Pull request by trevren11**
Made users page into a table and added some more information.

### CODE REVIEW

File: core/app.js [hunk 1]
```diff
@@ -296,7 +296,7 @@ app.use(_.post("/employees",function *(){
 app.use(_.get("/employees",function *(){
   var isAdmin = yield auth.isAdmin(this.session.user);
   var client = db();
-  var allAdmins = yield client.query("SELECT netid, email FROM users WHERE admin = TRUE;");
+  var allAdmins = yield client.query("SELECT netid, email, telephone, admin FROM users;");
```
Comment (dcravenus) [inline core/app.js]: Why is this allAdmins if it's all users?

Comment (trevren11) [inline core/app.js]: It will change in the next update.

Comment (dcravenus) [inline core/app.js]: If Ian is okay with that, I am okay with it.

Comment (webnard) [inline core/app.js]: :')

**Pull request merged**